rl-no-rpath.dpatch by  <doko@debian.org>

Don't set a runtime path for linking the shared libraries

Index: b/support/shobj-conf
===================================================================
--- a/support/shobj-conf	2010-04-10 12:05:07.742337113 +0000
+++ b/support/shobj-conf	2010-04-10 12:12:39.752338214 +0000
@@ -114,7 +114,7 @@
 	SHOBJ_LD='${CC}'
 	SHOBJ_LDFLAGS='-shared -Wl,-soname,$@'
 
-	SHLIB_XLDFLAGS='-Wl,-rpath,$(libdir) -Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
+	SHLIB_XLDFLAGS='-Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
 	SHLIB_LIBVERSION='$(SHLIB_LIBSUFF).$(SHLIB_MAJOR)$(SHLIB_MINOR)'
 	;;
 
